ares-proxy/broker_test.go

```
@@ -200,7 +207,7 @@ func TestBroker_PollClearsQueue(t *testing.T) {
 	go func() {
 		ctx := context.Background()
 		testRequest := json.RawMessage(`{"test": "request"}`)
-		broker.SubmitRequest(ctx, testRequest)
+		broker.SubmitRequest(ctx, "/v1/chat/completions", testRequest)
```

golangci-lint (2.12.2)

[error] Error return value of `broker.SubmitRequest` is not checked

(errcheck)